buildkite/src/Monorepo.dhall

```diff
@@ -28,6 +28,7 @@ let jobs : List JobSpec.Type =
 let prefixCommands = [
   Cmd.run "git config --global http.sslCAInfo /etc/ssl/certs/ca-bundle.crt", -- Tell git where to find certs for https connections
   Cmd.run "git fetch origin", -- Freshen the cache
```

Comment:
Would this fetch be unnecessary when executing pipelines for a fork? If not required I would add a condition here. It would save us the burden to download the ~450MB repo.